libs/@guardian/source-react-components-development-kitchen/src/file-input/FileInput.tsx

```diff
@@ -87,6 +88,13 @@ export const FileInput: FC<FileInputProps> = ({
 				hideLabel={hideLabel}
 				cssOverrides={fontSizes[size]}
 			>
+				{maxFileSize && (
+					<SupportingText>
+						Please note, the maximum file size is{' '}
+						{getReadableFileSize(maxFileSize)}.
+					</SupportingText>
+				)}
```

Comment:
Do we still want to show this if there’s an error? It might feel quite redundant…

Reply:
Hmm i'm not sure. There is one error which relates to the max file size but others don't relate and are separate to this message.